File: src/test/java/no/nav/pensjon/innsyn/domain/source/HardcodedForstegangstjeneste.java
package no.nav.pensjon.innsyn.domain.source;

import no.nav.pensjon.innsyn.domain.Forstegangstjeneste;
import no.nav.pensjon.innsyn.source.EntityGetter;

import java.util.List;

public class HardcodedForstegangstjeneste implements EntityGetter<Forstegangstjeneste> {

    @Override
    public List<Forstegangstjeneste> getEntities() {
        return List.of(
                new Forstegangstjeneste(
                        "tjenestestart",
                        "dimittert",
                        "rapporttype",
                        "status",
                        "kilde"));
    }
}

File: src/test/kotlin/no/nav/pensjon/innsyn/domain/source/HardcodedForstegangstjeneste.kt
package no.nav.pensjon.innsyn.domain.source

import no.nav.pensjon.innsyn.domain.Forstegangstjeneste

class HardcodedForstegangstjeneste : EntityGetter<Forstegangstjeneste?> {
    override val entities: List<Forstegangstjeneste>
        get() = java.util.List.of(
                Forstegangstjeneste(
                        "tjenestestart",
                        "dimittert",
                        "rapporttype",
                        "status",
                        "kilde"))
}